README.md

```diff
@@ -9,33 +9,41 @@
 
 # ChilliCream GraphQL Platform
 
-The ChilliCream GraphQL Platform, at its core, is a new way to create powerful Backends. HotChocolate, our .NET GraphQL server, connects any service or data source and creates a cohesive service to offer your consumers a unified API.
+Welcome to the ChilliCream GraphQL Platform!
 
-![ChilliCream GraphQL Platform](https://chillicream.com/resources/chillicream-graphql-platform.svg)
+We help you, developers and companies, to leverage your APIs to the next level with GraphQL. Strongly-typed schemas that match your APIs a 100%. Efficient data fetching that reduces overall cost without extra effort. Consumer-friendly, declarative, self documented APIs that support you in your daily work to build powerful UIs effectively.
 
-## Product Overview
+Most of our products are open-source and right here in this repository.
+
+Everyone is welcome! Always remember to treat anyone respectful, no matter their gender, opinion, religion, or skin-tone. We're one world and together we're stronger!
+
+Join our **awesome community** on [Slack](https://slack.chillicream.com), if you would like to get in touch with us, need help, or just want to learn!
+
+## Our Products
 
 ### Hot Chocolate
 
-Our GraphQL server to create GraphQL endpoints and merge schemas.
+Hot Chocolate is the most efficient, feature-rich, open-source GraphQL server in the .NET ecosystem, that helps developers to build powerful GraphQL APIs and Gateways with ease.
 
 [Documentation](https://chillicream.com/docs/hotchocolate/)
 
 ### Banana Cake Pop
 
-Our GraphQL IDE to explore, request and analyze any GraphQL endpoint.
+Banana Cake Pop is an awesome, performant, feature-rich GraphQL IDE that helps developers and data scientists to explore, share, and test any GraphQL API.
+
+Banana Cake Pop can be installed as [Desktop App](https://bananacakepop.com), used as [Web App](https://eat.bananacakepop.com), which can be installed through your browser of choice as well, or used as Middleware on your GraphQL endpoint. Middlewares are available in [.NET](https://www.nuget.org/packages/HotChocolate.AspNetCore) and [NodeJS](https://www.npmjs.com/package/@chillicream/bananacakepop-express-middleware). More middlewares will follow.
 
-[Learn more](https://chillicream.com/docs/bananacakepop/)
+[Documentation](https://chillicream.com/docs/bananacakepop/)
 
 ### Strawberry Shake
 
-Our GraphQL client to fetch data from any GraphQL endpoint.
+Strawberry Shake is an incredible, open-source GraphQL client for the .NET ecosystem, that helps developers to build awesome UIs in Blazor, Maui, and more.
 
 [Documentation](https://chillicream.com/docs/strawberryshake/)
 
 ### Green Donut
```

Comment:
Should we still mention Green Donut separately? I would just consider it part of Hot Chocolate.

Reply:
Good question. Honestly not sure. But we have still the package which is called GreenDonut, right?

Reply:
We do, but I imagine there are very few people who install it manually. since it comes with HotChocolate.AspNetCore and it's documentation is part of Hot Chocolate now.

Reply:
Yeah, lets mention it here, because the readme is for developers that want to get into our codebase. For docs I agree not to mention it.